### ISSUE TITLE
CSV Parsing: utf-8 issues

### ISSUE THREAD

**Issue by m1geo** [Joeskaife/JLCQT#11]
I get this error when parsing the JLCPCB CSV into database:

```
george@shannon:~/JLCQT$ python3 jlcqt.py
failed to get the current screen resources
QXcbConnection: XCB error: 170 (Unknown), sequence: 170, resource id: 90, major code: 146 (Unknown), minor code: 20
Traceback (most recent call last):
  File "jlcqt.py", line 373, in convertProcedure
    row_count = sum(1 for line in csvFile)
  File "jlcqt.py", line 373, in <genexpr>
    row_count = sum(1 for line in csvFile)
  File "/usr/lib/python3.5/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa6 in position 222: invalid start byte
Aborted
george@shannon:~/JLCQT$
```

Adding `try` and `expect UnicodeDecodeError` doesn't really help this issue.

**Comment by m1geo** [comment]
Adding `encoding='utf-8'` to the `open(...)` call doesn't help either. It's broken UTF-8 that's the issue, I think.

**Comment by m1geo** [comment]
https://stackoverflow.com/questions/904041/reading-a-utf8-csv-file-with-python may help?

**Comment by m1geo** [comment]
Fixed in #12 by adding `encoding='ISO8859'` to the `open(...)` call - I guessed the wrong encoding based on misleading error ;)
